### DISCUSSION TITLE
AWS S3 with Supabase Storage

### DISCUSSION THREAD

**Discussion by okadts** [supabase/supabase#6328] [Questions] [answered]
Any option to connect AWS S3 to Supabase Storage
Like the upload and file manager is using Supabase Storage but the files are saved to AWS S3
Thank you

**Comment by GaryAustin1** [comment, marked as answer]
If you are setting up your own server, yes. https://github.com/supabase/storage-api shows setup steps for S3.
For hosted Supabase currently uses S3 for storage of the files.
